samples/bluetooth/peripheral_uart/Kconfig

```diff
@@ -27,4 +27,10 @@ config BT_GATT_NUS_SECURITY_ENABLED
 	help
 	  "Enable BLE security for the UART service"
 
+config BT_GATT_NUS_UART_RX_WAIT_TIME
+	int "Wait for UART RX complete event time"
```

Comment:
Perhaps `Timeout for UART RX complete event` is clearer

Reply:
Changed